### DISCUSSION TITLE
Information after purshase

### DISCUSSION THREAD

**Discussion by fpicazo** [primefaces/community#3051] [PrimeReact Blocks]
hello i just bought Primereact but i am lost on how to use it, it is really confuse how to use the block and i try to navigate between primereact interface, primeblocks, primefaces etc and it is not possible to accede to a simple interface to use it or a tutorial, only option looks to download the file and have to start over from the project so it is not block use it is start to rebuild all instead of using blocks.

In the primeblock, i can only access VUE even in the subscription purchase it is written: PRIMEREACT, so how could i buy react if not available?
I hope you can send me some tutorial or how could i use the subscription as blocks.

**Comment by cagataycivici** [comment]
Hi,

Which version have you purchased?

The new one from PrimeBlocks;

https://primeblocks.org

or the old one from PrimeStore

https://blocks.primereact

via

https://primefaces.org/store

**Comment by fpicazo** [comment]
I am not sure, in the receipt it is PrimeBlocks for PrimeReact - Single
License (Qty: 1)

**Comment by cagataycivici** [comment]
What is the file name you downloaded, I don't see any Vue code in primeblocks-react primeblocks-react-3.3.0.zip for example. For getting started please see;

https://blocks.primereact.org/documentation
